spec/models/profile_spec.rb

```diff
@@ -299,9 +299,39 @@
 
       expect(profile).to be_active
     end
+
+    context 'when the initiating_sp is the IRS' do
+      it 'logs an attempt event' do
+        sp = create(:service_provider, :irs)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
+        )
+        expect(profile.initiating_service_provider.irs_attempts_api_enabled?).to be_truthy
+
+        expect_any_instance_of(IrsAttemptsApi::Tracker).to receive(:fraud_review_adjudicated).
+          with(determination: 'activated')
+        profile.activate_after_passing_review
+      end
+    end
+
+    context 'when the initiating_sp is not the IRS' do
+      it 'does not log an attempt event' do
+        sp = create(:service_provider)
+        profile = create(
+          :profile, user: user, active: false, fraud_review_pending: true,
+                    initiating_service_provider: sp
```

Comment:
Ditto. Is this actually a project norm?

Aside: I spent entirely too long trying to DRY this up only to make it a bigger mess, and backed that out. I'm still not thrilled with how this came out.